scripts/update_api_description.sh

```diff
@@ -17,7 +17,7 @@ function main() {
 
     mkdir -p "${OUT_DIR}"
 
-    go run cmd/describe-api/main.go > "${OUT_FILE}"
+    go run cmd/call-api-endpoint/main.go metadata/describe > "${OUT_FILE}"
```

Comment:
Just an FYI, this will change soon.

It's a (non-competitive) race between @OliverLok (#129) and this for who will need to change this line.